server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3082,6 +3082,10 @@ public TrafficTypeResponse createTrafficTypeResponse(PhysicalNetworkTrafficType
         PhysicalNetwork pnet = ApiDBUtils.findPhysicalNetworkById(result.getPhysicalNetworkId());
         if (pnet != null) {
             response.setPhysicalNetworkId(pnet.getUuid());
+            // using PhysicalNetworkResponse only to convert isolation methods to string
+            PhysicalNetworkResponse pnetResponse = new PhysicalNetworkResponse();
+            pnetResponse.setIsolationMethods(pnet.getIsolationMethods());
+            response.setIsolationMethods(pnetResponse.getIsolationMethods());
```

Comment:
this seems to me to indicate that the functionality should be extracted from `PhysicalNetworkResponse` and be put in a utility.

Reply:
@DaanHoogland do you think moving this logic to `PhysicalNetwork`, by overloading the getIsolationMethods method. What exists right now is `List<String> getIsolationMethods()`, I'm talking about adding a `String getIsolationMethods()` as well, that returns csv.

I'm not sure why `String.join` isn't used either, I don't think a separate utility is needed.

Reply:
@madhukar93, I am not sure about the dependency tree for responses but importing a response for the sole purpose of getting a field well formatted for another response seems a bit dodgy to me. 

Overloading with only a different return value is not going to work.

I think a dependency for both `PhysicalNetworkResponse` and `TrafficTypeResponse` should be found or created and the logic moved there.

Just calling a `String.join()` would work as well, but a common method to use in all places that need this formatting.